runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -413,6 +413,7 @@ createCrossLanguageValidatesRunnerTask(
   classpath: configurations.validatesRunner,
   numParallelTests: Integer.MAX_VALUE,
   needsSdkLocation: true,
+  semiPersistDir: "/var/opt/google",
```

Comment:
Why do we need to override this here?

Reply:
We use different semi persistant dirs for open-source runners and Dataflow runner (don't know why).